src/components/areaMenu/AposAreaMenu.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <div class="apos-area-menu" :class="{'apos-area-menu--grouped': groupedMenus, 'is-focused': groupIsFocused}">
-    <AposContextMenu tipAlignment="center" :modifiers="['unpadded']" >
+    <AposContextMenu tip-alignment="center" :modifiers="['unpadded']">
```

Comment:
kebab case here translates to camel case in the props list.

Reply:
🥴